src/DonorProfiles/resources/js/app/tabs/donation-history/content.js

```diff
@@ -37,10 +38,15 @@ const Content = () => {
 					{ __( 'Donation', 'give' ) } #{ id }
 				</Heading>
 				<DonationReceipt donation={ donations[ id ] } />
-				<div className="give-donor-profile__donation-history-link">
+				<div className="give-donor-profile__donation-history-footer">
 					<Link to="/donation-history">
 						<FontAwesomeIcon icon="arrow-left" /> { __( 'Back to Donation History', 'give' ) }
 					</Link>
+					{ donations[ id ].payment.pdfReceiptUrl.length && (
+						<Button icon="file-pdf" onClick={ () => window.location = donations[ id ].payment.pdfReceiptUrl }>
```

Comment:
Is there a particular reason why we are using `window.location` in a `onClick` as opposed to using a hyperlink with an `href`?

Reply:
My concern is that this is not accessible. Screen readers won't find this because it isn't an HTML link.

Reply:
Good thinking, I've gone ahead and incorporated this suggestion here: https://github.com/impress-org/givewp/pull/5613/commits/3d09eb3eaa897a2eb249f49662e86934669d2369